src/System.Management.Automation/FormatAndOutput/DefaultFormatters/DotNetTypes_format_ps1xml.cs

```diff
@@ -478,6 +486,46 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Version()
                 .EndTable());
         }
 
+        private static IEnumerable<FormatViewDefinition> ViewsOf_System_Version_With_Label()
+        {
+            yield return new FormatViewDefinition("System.Version",
+                TableControl.Create()
+                    .AddHeader(width: 6)
+                    .AddHeader(width: 6)
+                    .AddHeader(width: 6)
+                    .AddHeader(width: 8)
+                    .AddHeader(width: 26)
+                    .AddHeader(width: 27)
+                    .StartRowDefinition()
+                        .AddPropertyColumn("Major")
+                        .AddPropertyColumn("Minor")
+                        .AddPropertyColumn("Build")
+                        .AddPropertyColumn("Revision")
+                        .AddPropertyColumn("PSSemanticVersionPreLabel")
```

Comment:
Would it make sense to change this property name to `PSSemanticVersionPrereleaseLabel`?

Reply:
Looks very long 😕

Reply:
Maybe ` PSSemVerPreReleaseLabel `?

Reply:
That looks good to me, especially `semver` is the type accelerator for `SemanticVersion`.

Reply:
Fixed.